graphql/ui_plugin_build_baron.go

```diff
@@ -76,6 +76,15 @@ func BbFileTicket(context context.Context, taskId string, execution int) (bool,
 	return taskNotFound, nil
 }
 
+func IsWebhookConfigured(t *task.Task) bool {
```

Comment:
FYI you're setting a lot of variables you don't need here, for the sake of keeping the line short. I realize that `BbGetConfig(env.Settings())[t.Project].TaskAnnotationSettings.FileTicketWebHook.Endpoint` is a bit long, but it may be possible to split into fewer sections? I personally find that splitting it this much is more confusing; it makes the helper function look like more than it is, when it's really just grabbing a value from the settings. This is a style preference so you can keep it if you want but I thought I'd weigh in.

Reply:
That's fair, I'll consolidate it.